test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,25 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+# flags: --fast-parser
+f'foobar'
+
+[case testFStringTypecheckExpression]
+# flags: --fast-parser
+# flags: --python-version 3.6
+a: str
+a = 'foo'
+b: str
+b = f'{a}bar'
+b == 'foobar'
```

Comment:
I get `Unsupported left operand type for == ("str")` here, which I think is a red herring. Did you mean `=` instead of `==`?